Jellyfin.Plugin.Tvdb/Providers/TvdbMissingEpisodeProvider.cs

```diff
@@ -325,6 +325,12 @@ private async Task<IReadOnlyList<EpisodeRecord>> GetAllEpisodes(int tvdbId, stri
                         episodeQuery,
                         acceptedLanguage,
                         CancellationToken.None).ConfigureAwait(false);
+
+                    if (episodes.Data == null)
+                    {
+                        throw new TvDbServerException($"Episode Query returned null for tvdbId: {tvdbId}", -1);
```

Comment:
There's no reason to throw an exception just to catch it immediately after. Just log and return an empty array.

Reply:
Sure, guess that's a matter of taste. Changed in f23df75.